src/modules/compliance/src/lib/Procedure.cpp

```diff
@@ -5,52 +5,147 @@
 
 #include "Logging.h"
 
+#include <cassert>
 #include <parson.h>
-#include <sstream>
 
 namespace compliance
 {
+namespace
+{
+std::size_t SkipSpaces(const std::string& input, std::size_t pos)
+{
+    while (pos < input.size() && isspace(input[pos]))
```

Comment:
>pos < input.size() [](http://example.com/codeflow?start=11&length=18)

A new set of braces may be useful here, what do you think? One rule we try to follow is to never rely on compiler operator precedence.